figures/generate-figures.sh

```diff
@@ -32,176 +32,230 @@ mkdir -p pdfs
 Rscript --vanilla scripts/color_palettes.R
 Rscript -e "rmarkdown::render('mapping-histology-labels.Rmd', clean = TRUE, params = list(release = 'release-v21-20210820'))"
 
-################ Sample distribution
-# Run sample distribution analysis
-bash ${analyses_dir}/sample-distribution-analysis/run-sample-distribution.sh
 
-# TODO: Rscript for figure (related: https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1175)
+##### Run all analysis modules required for figures ---------------------------------------
 
-######################
-## Interaction plots
+# Run modules that cannot be run locally due to memory requirements
+if [ "$RUN_LOCAL" -lt "1" ]; then
+  bash ${analyses_dir}/focal-cn-file-preparation/run-prepare-cn.sh      # Figures 2 and S3
+  Rscript --vanilla scripts/supp-snv-callers-panels.R                   # Figure S2
+  Rscript --vanilla scripts/supp-tmb-compare-panels.R                   # Figure S2
+  bash ${analyses_dir}/copy_number_consensus_call/run_consensus_call.sh # Figure S3 (heatmap)
+fi
 
-# Create directory that will hold the relevant panel
-mkdir -p pdfs/fig3/panels
+# Run the `oncoprint-landscape` module shell script, for Figure 2 and S3
+bash ${analyses_dir}/oncoprint-landscape/run-oncoprint.sh
 
-# Run the main figure generation script
+# Run the interaction plots script for Figures 3 and S3
 bash ${analyses_dir}/interaction-plots/01-create-interaction-plots.sh
 
-# Copy the main figure to final directory
-cp ${analyses_dir}/interaction-plots/plots/combined_top50.pdf pdfs/fig3/panels/mutation_cooccurrence_figure.pdf
 
+# Run the chromothripsis module, which requires the chromosomal-instability module, for Figure 3
+bash ${analyses_dir}/chromosomal-instability/run_breakpoint_analysis.sh
+bash ${analyses_dir}/chromothripsis/run-chromothripsis.sh
 
-#####################
-## Chromothripsis
+# Run the mutational-signatures module for Figures 3 and S4
+# We only run the part of the module used in the manuscript (i.e., not de novo)
+Rscript --vanilla ${analyses_dir}/mutational-signatures/05-fit_cns_signatures.R
+Rscript -e "rmarkdown::render('${analyses_dir}/mutational-signatures/06-compare_cns_exposures.Rmd', clean = TRUE)"
+Rscript -e "rmarkdown::render('${analyses_dir}/mutational-signatures/07-plot_cns_fit.Rmd', clean = TRUE)"
+Rscript -e "rmarkdown::render('${analyses_dir}/mutational-signatures/08-explore_hypermutators.Rmd', clean = TRUE)"
 
-# The chromothripsis module uses breakpoint counts from this module
-bash ${analyses_dir}/chromosomal-instability/run_breakpoint_analysis.sh
 
-# Run the chromothripsis module
-bash ${analyses_dir}/chromothripsis/run-chromothripsis.sh
+# Run the collapse-rnaseq module, which is needed for telomerase, immune deconvolution, and GSVA modules
+bash ${analyses_dir}/collapse-rnaseq/run-collapse-rnaseq.sh 
```

Comment:
If #1394 goes in, this would need to be:
```suggestion
# Run the collapse-rnaseq module, which is needed for telomerase, immune deconvolution, and GSVA modules
OPENPBTA_TP53_FIGURES=1 bash ${analyses_dir}/collapse-rnaseq/run-collapse-rnaseq.sh 
```